### ISSUE TITLE
TLoZ Breath of the Wild: Occasional crash when opening camera

### ISSUE THREAD

**Issue by clackie1996** [yuzu-emu/yuzu#4612]
Yuzu Version: 363

System Specs: 
- Intel i5 8400 @ ~3.8GHz, 
- NVIDIA GeForce GTX 1060 6GB,
- 16GB DDR4 RAM

Repro Steps:
1. Play for a short while, making normal use of the camera feature.
2. After 10-15 minutes of play or so, open the camera.
3. Observe game crash.

Compressed log file attached.
[yuzu_log.zip](https://github.com/yuzu-emu/yuzu/files/5146890/yuzu_log.zip)

I cannot find a guide on obtaining a backtrace anywhere. If someone tells me how to, I can provide this also.

**Comment by clackie1996** [comment]
Related, game just crashed after taking a picture and closing the camera. It was about time for the camera to crash, so I saved, got into the camera without issue (surprising), but then after taking the picture and closing the camera, the game crashed. Checked Task Manager and everything had spiked right to 100% right as the game crashed.

Compressed log attached.
[yuzu_log.zip](https://github.com/yuzu-emu/yuzu/files/5147058/yuzu_log.zip)
